### ISSUE TITLE
Support of -intf -impl -intf-suffix

### ISSUE THREAD

**Issue by jordwalke** [the-lambda-church/merlin#470]
I have a build system that uses a file extension to control which preprocessor is applied. I was invoking some merlin commands (`Locate`) which should have been able to find the file, but it failed with:
```
'MyModule' seems to originate from 'MyModule' whose ML file could not be found
``` 

Merlin `FLG`s are probably not the appropriate place to specify the `-intf/-impl/-intf-suffix` since they are supplied *per input file*.

Was I merely using the API incorrectly, or is it true that custom extensions are not supported?

**Comment by trefis** [comment]
This was fixed by #474 , wasn't it?

**Comment by jordwalke** [comment]
Yes.
